components/sidebar.jsx

```diff
@@ -71,7 +80,7 @@ var Sidebar = React.createClass({
           <Link to="home">
             <img src="/img/nav/mozilla-wordmark-white.svg" alt="Webmaker logo" className="moz-logo"/>
           </Link>
-          <span aria-label="toggle" role="button" onKeyUp={this.handleHamburgerClick} onKeyDown={this.handleHamburgerClick} className="glyphicon glyphicon-menu-hamburger hidden-lg hidden-md"
+          <span aria-label="toggle" role="button" onKeyDown={this.handleHamburgerKeyDown} className="glyphicon glyphicon-menu-hamburger hidden-lg hidden-md"
```

Comment:
Can we make this span a button? Then the onClick should do exactly what we need without needing to filter input and we can then also remove role="button" I think.